src/env.ts

```diff
@@ -32,6 +32,35 @@ export function defineEnv(opts: CreateEnvOptions = {}): {
 
   const resolvedEnv = env(...presets);
 
+  if (opts.resolve) {
+    const resolveOpts: ResolveOptions = {
+      url: [
+        ...(opts.resolve === true ? [] : opts.resolve.paths || []),
```

Comment:
nit
```suggestion
        ...(opts.resolve === true ? [] : opts.resolve.paths ?? []),
```

Reply:
nullish is invalid (like `0` or `false`) it will cause runtime error when spreading in same line.